scripts/uitwerkingen.js

```diff
@@ -32,7 +35,7 @@ switch (userInput) {
     default:
         console.error("ongeldige keuze")
 }
-/*/
+
```

Comment:
Opdracht 3a zijn de iteraties van job titels in de string geschreven, die had buiten de string moeten zijn. Na afsluiten haakje van prompt functie ontbreekt een ';'.

Reply:
Bij cases heb je de laatste iteratie en '+' in een string geschreven. 
Dit had buiten de string moeten zijn.